util/regexp-assemble/data/934100.data

```diff
@@ -24,9 +24,150 @@
 
 _\$\$ND_FUNC\$\$_
 __js_function
-\beval\s*\(
-String\s*\.\s*fromCharCode
-function\s*\(\s*\)\s*{
+\beval\(
+String\.fromCharCode
+function\(\){
 new\s+Function\s*\(
 this\.constructor
-module\.exports\s*=
+module\.exports=
+
+
+##!> assemble
+\[[\'\"\`]
+##!=< js-prop-start
+##!<
+
+##!> assemble
+[\'\"\`]\]
+##!=< js-prop-finish
+##!<
+
+##!> assemble
+\(
+\.call\(
+##!=< js-call
+##!<
+
+
+##! "process" payloads
+##!> assemble
+process
+##!=>
+
+##! All method names
+##!> assemble
+
+##!=< func
+##!<
+
+##! All prop names
+##!> assemble
+env
+##!=< prop
+##!<
+
+##! Match method calls via their usual syntax: foo.bar()
+##!> assemble
+##! NOTE: Uncomment lines below if methods are defined
+##! \.
+##!=>
+##!    ##!> assemble
```

Comment:
No need to use nested processors for concatenating for borth `func` and `js-call` (will not repeat for same issues below).